lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class NetworkTokenizationCreditCard < CreditCard
+      # A +NetworkTokenizationCreditCard+ object represents a tokenized credit card 
+      # using the EMV Network Tokenization specification, http://www.emvco.com/specifications.aspx?id=263.
+      #
+      # It includes all fields of the +CreditCard+ class with additional fields for 
+      # verification data that must be given to gateways through existing fields (3DS / EMV).
+      #
+      # The only tested usage of this at the moment is with an Apple Pay decrypted PKPaymentToken, 
+      # https://developer.apple.com/library/ios/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html 
```

Comment:
I actually think what you already have in ``PaymentToken`` makes a lot of sense already:

```
   # Base class representation of cryptographic payment data tokens that may be used for EMV-style transactions
    # like Apple Pay. Payment data may be transmitted via any data type, and may also be padded
    # with metadata specific to the cryptographer. This metadata should be parsed and interpreted in concrete
    # implementations of your given cryptographer. Like credit cards, you must also return a string representing
    # the token's type, like 'apple_pay' or 'stripe' should your target payment gateway process these tokens.
```

what do you think?